detekt.yml

```diff
@@ -1,2 +1,6 @@
 autoCorrect: false
 failFast: false
+
+style:
```

Comment:
Can we add a comment to remove this before initial release since this is temporary while we are allowing TODO comments

Reply:
Done.